controllers/release/adapter_test.go

```diff
@@ -149,6 +154,8 @@ var _ = Describe("Release adapter", Ordered, func() {
 					},
 				},
 			})
+			adapter.releaseServiceConfig = releaseServiceConfig
```

Comment:
@davidmogar thoughts on these lines? I could change `createReleaseAndAdapter` to do this instead of doing it manually in the tests that require it